### PR TITLE
Adding documentation to undocumented attributes

### PR DESCRIPTION
**Pull request by nathancarter**
Hopefully these docs can help others who browse to this page and want to know what the various stats are and what they mean.  Not all meanings were clear to me from data I've inspected, so I'm leaving some unspecified.  Obviously anyone can come along later and propose a change to add info I didn't know how to add.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -467,10 +499,32 @@ For fields in `participant` such as `actor` or `itemGrants`, server will return
...
-- `.shardId`
-- `.stats`
-- `.titleId`
-- `.skillTier`
-- `.karmaLevel`
-- `.createdAt`
+- `.name` - a string, the player's IGN
+- `.shardId` - region from which the match was fetched, in lower case (e.g., `"na"`)
+- `.stats` - an object with the following attributes:
+   - `elo_earned_season_4` - meaning unclear
```
Comment (PierreAndreis) [inline README.md]: Those values are all deprecated, they were replaced by rankPoints

File: README.md [hunk 2]
```diff
@@ -467,10 +499,32 @@ For fields in `participant` such as `actor` or `itemGrants`, server will return
...
+   - `elo_earned_season_8` - meaning unclear
+   - `elo_earned_season_9` - meaning unclear
+   - `gamesPlayed` - object whose keys are various match types (aral, blitz, blitz_rounds, casual, casual_5v5, ranked, etc.) and whose values are the number of times the player has played that match type
+   - `guildTag` - a string of up to 4 characters, the player's guild tag, or the empty string if the player has no guild
+   - `karmaLevel` - an integer, one of the three [listed here](src/models/resources/karma.js)
+   - `level` - an integer, the level of the player (in the sense of gaining experience post-match to level up your account)
+   - `lifetimeGold` - meaning unclear
+   - `lossStreak` - number of matches the player has lost in a row, or 0 if the last match was a win (but sometimes this number and `winStreak` are both zero, impossibly)
+   - `played` - meaning unclear
+   - `played_aral` - meaning unclear
```
Comment (PierreAndreis) [inline README.md]: same for these

Comment (PierreAndreis) [reply README.md]: they were replaced by gamesPlayed

File: README.md [hunk 3]
```diff
@@ -451,10 +460,33 @@ For fields in `participant` such as `actor` or `itemGrants`, server will return
...
+   - `deaths` - an integer, number of times the player died (the second number in KDA)
+   - `farm` - an integer, meaning unclear, possibly number of monsters/minions killed
+   - `firstAfkTime` - an integer, the number of seconds into the match the player first went AFK, or -1 if they never did
+   - `gold` - a number (not usually an integer), total gold earned by this player throughout the match
+   - `goldMineCaptures` - an integer, the number of times the player killed the gold miner
+   - `itemGrants` - an object whose keys are the names of the items the player purchased, and whose values are the number of times the player purchased that item; example: `{"Weapon Blade":2,"Six Sins":1,"Heavy Steel":1,"Sorrowblade":1,...}`
+   - `itemSells` - similar in structure to the previous, but for selling items
+   - `itemUses` - an object whose keys are the names of items the player used, and whose values are the number of times the player used the item; example: `{"Travel Boots":3,"Scout Cam":3,...}`
+   - `items` - array of strings, all items the player possessed at the end of the match
+   - `jungleKills` - meaning uncertain
```
Comment (PierreAndreis) [inline README.md]: Jungle camps

File: README.md [hunk 4]
```diff
@@ -413,13 +413,15 @@ For fields in `participant` such as `actor` or `itemGrants`, server will return
...
 
 - `.assets` - Array of [Asset](#assetModel)
-- `.createdAt`
-- `.duration`
-- `.gameMode`
-- `.patchVersion`
-- `.shardId`
-- `.stats`
-- `.titleId`
+- `.createdAt` - a string, the match timestamp, suitable for passing to JavaScript `Date` constructor
```
Comment (PierreAndreis) [inline README.md]: This would be also good to highlight that it's the time on which the match was started, not when it ended

**Comment by PierreAndreis** [comment]
It looks really good to me, really helpful

**Comment by nathancarter** [comment]
What's the best way for me to incorporate your corrections, @PierreAndreis ?  Before now, I've never needed to modify a GitHub PR, so I'm not familiar with the process.

**Comment by PierreAndreis** [comment]
@nathancarter any commits to the origin of this PR will show here. In this case, it's the branch `patch-1` on your fork

If you are using the web UI, just modify this file https://github.com/nathancarter/vainglory/blob/patch-1/README.md

**Comment by nathancarter** [comment]
OK...swamped at actual job at the moment...will get back to this...thanks!

**Comment by nathancarter** [comment]
The changes you suggested have been made.  Thank you!

**Comment by seripap** [comment]
:+1:

**Comment by nathancarter** [comment]
Great, thanks, glad to be of some assistance.  This repo is super helpful!

**Pull request merged**